test/com/google/javascript/jscomp/GoldenFileComparer.java

```diff
@@ -104,6 +105,33 @@ private static void compileAndCompare(
     compare(compiledSource, referenceSource);
   }
 
+  /**
+   * Compile sourceFileName with given CompilerOptions and check that all lines of code from
+   * goldenFileName are present in compiled sourceFile.
+   */
```

Comment:
Would be good to mention use case of this function: when output contains polyfills and polyfills should be ignored.

Reply:
Updated.